pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/tasks/PyTestTask.groovy

```diff
@@ -47,8 +46,6 @@ class PyTestTask extends AbstractPythonTestSourceDefaultTask {
     void processResults(ExecResult execResult) {
         if (execResult.exitValue == NO_TESTS_COLLECTED_ERRNO) {
             logger.warn("***** WARNING: You did not write any tests! *****")
-        } else {
```

Comment:
Do we still need `else` and call to `super.processResults(execResult)` or some other form of action here when `exitValue` is non-zero and not `NO_TESTS_COLLECTED_ERRNO`? We need to raise an exception at least and `assertNormalExitValue` was ensuring that. I think it's still necessary.